src/middleware/externalRequests.js

```diff
@@ -31,17 +31,22 @@ const performRequests = requests => {
   })
 }
 
-const prepareLookupRequests = ctx => {
+function prepareLookupRequests(ctx) {
   const requests = Object.assign({}, ctx.state.metaData.requests)
   if (requests && requests.lookup && requests.lookup.length > 0) {
```

Comment:
I dont think the first condition is necessary here as `requests` will always be defined. By using the object.assign, an empty object will be created if the `metaData.requests` doesnt exist

Reply:
Done in https://github.com/jembi/openhim-mediator-mapping/pull/37/commits/28787dbefd3e9d8b7adfc4663692c8c87aacbf96